payments_service/braintree/tests/test_forms.py

```diff
@@ -90,6 +90,18 @@ def test_missing_pay_method(self):
             form.errors, '__all__',
             msg='Either pay_method_nonce or pay_method_uri can be submitted')
 
+    def test_cannot_pay_with_another_users_pay_method(self):
+        self.user_owns_resource.return_value = False
+
+        form = self.submit(
+            expect_errors=True,
+            overrides=dict(pay_method_uri='/someone/elses/paymethod/123',
+                           pay_method_nonce=None)
+        )
+        self.assert_form_error(
+            form.errors, 'pay_method_uri',
+            msg='paymethod by URI.*belongs to another user')
```

Comment:
yeah. This way the test doesn't depend on non-existant pay methods -- that's another part of the message for another test.